jib-core/src/test/java/com/google/cloud/tools/jib/frontend/MainClassFinderTest.java

```diff
@@ -126,4 +126,15 @@ public void testFindMainClass_innerClasses() throws URISyntaxException, IOExcept
         mainClassFinderResult.getFoundMainClass(),
         CoreMatchers.containsString("HelloWorld$InnerClass"));
   }
+
+  @Test
+  public void testMainClass_varargs() throws URISyntaxException, IOException {
+    Path rootDirectory =
+        Paths.get(Resources.getResource("core/class-finder-tests/varargs").toURI());
+    MainClassFinder.Result mainClassFinderResult =
+        new MainClassFinder(new DirectoryWalker(rootDirectory).walk(), mockEventDispatcher).find();
+    Assert.assertSame(Type.MAIN_CLASS_FOUND, mainClassFinderResult.getType());
```

Comment:
Can't `assertEquals` work?

Reply:
It can, but we use "same" in most of the other `MainClassFinder` tests (`assertSame` tests using `==`, `assertEquals` tests using `.equals()`)